tests/unit_tests/services/test_dds.py

```diff
@@ -73,122 +56,237 @@ def wait_as_coroutine(x):
                 staging_service=self.mock_staging_service,
                 staging_dir='/foo/bar/staging_dir',
                 delivery_repo=self.mock_delivery_repo,
-                dds_project_repo=self.mock_dds_project_repo,
                 session_factory=self.mock_session_factory,
-                dds_conf=self.mock_dds_config
+                dds_conf=self.mock_dds_config,
```

Comment:
Good question! I think it usually makes the diff a bit neater and makes it simpler to add/reorder arguments, e.g. avoid re-running the tests because I added a new arguments and forgot to add the comma to the one before :sweat_smile: 

It's also fine by PEP8: https://peps.python.org/pep-0008/#when-to-use-trailing-commas

Reply:
Ah that's true, did not know that it was allowed, TIL :)